### ISSUE TITLE
Footer Information

### ISSUE THREAD

**Issue by joycekimdesigns** [factn/Resilience-Design#47]
Find out the details that will be needed for the footer:
- parent website
- registered legal text
- logos to include
- etc...

**Comment by Erioldoesdesign** [comment]
**Must Have**
- A link to Resilience: Mutual aid Open Collective page: https://opencollective.com/resilience with the Open Collective logo or banner: https://docs.opencollective.com/help/collectives/widgets
- A link to our main, parent website: https://mutualaid.world/
- Our organisation logo (new logo pending)
- Our github + logo for github: https://github.com/factn/resilience-app
- A contact us link: https://mutualaid.world/contact/ (this is tricky because we don't want users thinking they're contacting feed folks when they are contacting mutual.aid 🤔 

**Nice to have**
- Link to 'how to volunteer for the parent org': https://mutualaid.world/volunteer/
- Our twitter: https://twitter.com/Resilience_App
- Our linked in:https://www.linkedin.com/company/mutualaid-world/

**Comment by joycekimdesigns** [comment]
3/26/21 Meeting Recap 
Participants: Eriol & Joyce 

Eriol explained Footer details, where to find info

**Comment by joycekimdesigns** [comment]
I categorized the list Eriol provided for the footer information into the following:
<img width="1138" alt="Screen Shot 2021-04-01 at 12 41 43 AM" src="https://user-images.githubusercontent.com/78825537/113261949-eaf20500-9284-11eb-988b-df59f8bf7b9f.png">

Considering the smallest device users would be using to look at the footer information, I thought having different icons/images to link to different places would not give the footer a uniform look and the messiness would eventually reflect badly on the credibility of MutualAid. I came up with four layouts for the footer.

**Comment by joycekimdesigns** [comment]
Design A:
![21 - Footer A](https://user-images.githubusercontent.com/78825537/113262683-bc285e80-9285-11eb-93cf-3fb5a936666f.png)

**Comment by joycekimdesigns** [comment]
Design B:
![21 - Footer B](https://user-images.githubusercontent.com/78825537/113262721-c64a5d00-9285-11eb-8502-ddcd57fd758a.png)

**Comment by joycekimdesigns** [comment]
Design C:
![21 - Footer C](https://user-images.githubusercontent.com/78825537/113262766-d5310f80-9285-11eb-827b-356c5cd811a5.png)

**Comment by joycekimdesigns** [comment]
Design D:
![21 - Footer D](https://user-images.githubusercontent.com/78825537/113262799-deba7780-9285-11eb-9118-aedbdecf643f.png)

**Comment by thoughtsky** [comment]
Great! I think B has everything here. Thoughts on having a dropdown for "Contribute"? This would tidy it up and only show the child items on demand.

**Comment by Erioldoesdesign** [comment]
Agreed with B but I don't think we need a drop down - the indent works for me for hierarchy. When we get more links we can think about a dropdown interaction.
